emscripten/webarkitJsfeat.cpp

```diff
@@ -168,14 +169,17 @@ void train_orb_pattern_internal(const char* filename) {
   auto i_u8_size = jpegImage->xsize * jpegImage->ysize * jpegImage->nc;
   Array<u_char> i_u8(jpegImage->image, jpegImage->image + i_u8_size);
   std::unique_ptr<Matrix_t> img_u8 = std::make_unique<Matrix_t>(jpegImage->xsize, jpegImage->ysize, ComboTypes::U8C4_t, i_u8);
-
-  imgproc.resample(img_u8.get(), lev0_img.get(), new_width, new_height);
-
+  
+  JSLOGi("Resampling image...");
+  
```

Comment:
Resampling is not needed in our case, because we provide our image with the right size. The code was taken by the jsfeat sample_orb example and in that case we simply resampled the image taken by the canvas(webcam) to a smaller size. Anyway the resample function has some issues, infact the log console "Image resampled, starting pyrmaid now..." can not be printed with this function enabled. (just comment out and recompile to test)